Makefile

```diff
@@ -1,25 +1,183 @@
-NAME:=vrt
-LICENSE:="OSI Approved :: Apache Software License v2.0"
-AUTHOR:="Afiniate, Inc."
-ORGANIZATION:="afiniate"
-HOMEPAGE:="https://github.com/afiniate/vrt"
+# =============================================================================
+# Build Management Knobs
+# =============================================================================
 
-DEV_REPO:="git@github.com:afiniate/vrt.git"
-BUG_REPORTS:="https://github.com/afiniate/vrt/issues"
+PARALLEL_JOBS ?= 2
```

Comment:
So, not for this PR, but I'd like a way for something like this to be set to a default, or pulled from somewhere. Maybe we build in a way to get default params from `trv` or we find a sane way to probe the system?